### DISCUSSION TITLE
how do i use

### DISCUSSION THREAD

**Discussion by PINKeHamton** [expressobits/character-controller#15] [General]
I am having trouble using this in my game i don't think I am meant to drag it from the file manager to the working window. 
if i am missing something pleas tell me.

**Comment by PINKeHamton** [comment]
it was a lighting issue

**Comment by technificentConsulting** [comment]
How do you use the non FPS controller?

**Reply by PINKeHamton** [reply]
im sorry I stopped using Godot and stated using SDL2 and C

**Reply by technificentConsulting** [reply]
No worries, thanks for responding and converting this to C# and good luck on your current project!

**Reply by scriptsengineer** [reply]
> How do you use the non FPS controller?

Hello! I haven't really used him as an NPC yet, but the idea is very simple, just use the FPSCOntroller or controller giving inputs coming from your AI script or another that is part of the NPC.

**Reply by technificentConsulting** [reply]
This is what I've been attempting.  I think I may have a working solution but wasn't sure if there was a "built in" way to do it by toggling one of the exported vars or something since it says in the features ...and non FPS Character Controller.
